### ISSUE TITLE
Is this relevant to arrays ?

### ISSUE THREAD

**Issue by netkap** [kunal-kushwaha/DSA-Bootcamp-Java#898]
In arrays assignment the question number 10 https://leetcode.com/problems/check-if-the-sentence-is-pangram/ is related to string as per my observation . However I am still learning and it might be related to array.

**Comment by ishangaur07** [comment]
Yeah, Its a string based question

**Comment by FarhaadKhan1706** [comment]
Yes you can use an array to store strings  like this - String[] words = new String[5];
